snomed/com.b2international.snowowl.validation.snomed/src/test/java/com/b2international/snowowl/validation/snomed/BaseGenericValidationRuleTest.java

```diff
@@ -92,7 +94,8 @@ protected void configureContext(Builder context) {
 	@Override
 	protected void configureValidationRequest(ValidateRequestBuilder req) {
 		req.setRuleParameters(Map.of(
-			ValidationConfiguration.IS_UNPUBLISHED_ONLY, effectiveTime == EffectiveTimes.UNSET_EFFECTIVE_TIME
+			ValidationConfiguration.IS_UNPUBLISHED_ONLY, effectiveTime == EffectiveTimes.UNSET_EFFECTIVE_TIME,
+			ValidationConfiguration.WORKING_MODULES, "<<" + Concepts.UK_DRUG_EXTENSION_MODULE
```

Comment:
You can also use a customer independent module ID for the unit tests. We have our own module called `MODULE_B2I_EXTENSION`

Reply:
Done